benchmarks/tcpdump-4.9.0/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -ex
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+. $(dirname $0)/../common.sh
+
+build_lib() {
+  rm -rf BUILD
+  cp -rf SRC BUILD
+  # we build and install bc was having issues with autogen.sh of dbus
+  # makes building the rest easier too..
+  (cd BUILD/libexpat/expat && ./buildconf.sh && ./configure && make &&
+   make install)
+  (cd BUILD/libdbus && 
+    CFLAGS="${CFLAGS} -Wno-error" ./autogen.sh --disable-xml-docs --disable-doxygen-docs --disable-ducktype-docs &&
+    make && make install)
+  (cd BUILD/libpcap && ./configure && make &&
+   cd ../tcpdump && ./configure && make)
+}
+
+#
+# tcpdump really wants libpcap to share the same parent directory.
```

Comment:
Can you please drop the empty comment lines below and above this line?

Reply:
Fixed